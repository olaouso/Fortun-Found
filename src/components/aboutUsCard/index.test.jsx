import React from "react";
import renderer from "react-test-renderer";
import AboutUsCard from "./";

it("renders AboutUsCard correctly", () => {
  const tree = renderer.create(<AboutUsCard />).toJSON();
  expect(tree).toMatchSnapshot();
});
