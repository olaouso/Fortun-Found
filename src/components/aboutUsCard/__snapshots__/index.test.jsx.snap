// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AboutUsCard correctly 1`] = `
<div
  className="col-md-4"
>
  <h4
    className="cardTitle"
  />
  <p
    className="cardParagraph"
  />
</div>
`;
